Makefile

```diff
@@ -57,10 +57,11 @@ run-ios: | start
 		@exit 1; \
 	fi
 
+run-ios: | check-device-ios start
 	@echo Running iOS app in development
 	@react-native run-ios --simulator="${SIMULATOR}"
```

Comment:
Shouldn't both ios and android skip the packager if it's ran by `make start`?

Reply:
the packager will not run, so this is fine

Reply:
Then why add --no-packager in `make run-android`?

Reply:
cause if the packager wasn't running before there seems to be a race condition and it ends up running twice

Reply:
Ah okay, so it's only a problem with the Android one. RN really does not work well with Android